src/hx711-element.ts

```diff
@@ -0,0 +1,491 @@
+              id="Rectangle-3"
+              stroke="#FFFFFF"
+              stroke-width="2"
+              x="25"
+              y="1"
+              width="26"
+              height="147"
+            ></rect>
+            <rect
+              id="Rectangle-3"
+              stroke="#FFFFFF"
+              stroke-width="2"
+              x="270"
+              y="22"
+              width="25"
+              height="102"
+            ></rect>
+            <text
+              id="GND"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="14"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="297.444824" y="42">GND</tspan>
+            </text>
+            <text
+              id="DT"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="14"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="297.168945" y="66">DT</tspan>
+            </text>
+            <text
+              id="SCK"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="14"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="297.220703" y="91">SCK</tspan>
+            </text>
+            <text
+              id="VCC"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="14"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="297.220703" y="116">VCC</tspan>
+            </text>
+            <text
+              id="A+"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="16"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="0.05078125" y="94">A+</tspan>
+            </text>
+            <text
+              id="B-"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="16"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="0.05859375" y="117">B-</tspan>
+            </text>
+            <text
+              id="B+"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="16"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="0.05078125" y="140">B+</tspan>
+            </text>
+            <text
+              id="E-"
+              font-family="Arial-BoldItalicMT, Arial"
+              font-size="16"
+              font-style="italic"
+              font-weight="bold"
+              fill="#FBFBFB"
+            >
+              <tspan x="0" y="45">E-</tspan>
+            </text>
```

Comment:
All the <text> have the same style, and can be combined into a single `<text>` element with multiple `<tspan>` elements. 

e.g. https://github.com/wokwi/wokwi-elements/blob/c40b1d7482133dc2b418a5e3a05a179b9d7fbc01/src/esp32-devkit-v1-element.ts#L193-L224